src/GraphQL.Tests/Execution/VariablesTests.cs

```diff
@@ -187,8 +187,7 @@ public void fail_on_incorrect_value()
             """;
 
         var result = AssertQueryWithErrors(query, null, rules: Enumerable.Empty<IValidationRule>(), expectedErrorCount: 1, executed: false);
-        result.Errors[0].Message.ShouldBe("Error trying to resolve field 'fieldWithObjectInput'.");
-        result.Errors[0].InnerException.Message.ShouldStartWith("Expected object value for 'TestInputObject', found not an object '[\"foo\", \"bar\", \"baz\"]'.");
+        result.Errors[0].Message.ShouldBe("""Invalid literal for field 'fieldWithObjectInput' argument 'input'. Expected object value for 'TestInputObject', found not an object '["foo", "bar", "baz"]'.""");
```

Comment:
Invalid literal for field 'fieldWithObjectInput' argument 'input'.
->
Invalid literal for argument 'input' of field 'fieldWithObjectInput'.

Reply:
Updated throughout.